ofctrl/fgraphFlow.go

```diff
@@ -447,7 +447,7 @@ func (self *Flow) xlateMatch() openflow13.Match {
 		ofMatch.AddField(*ctMarkField)
 	}
 
-	if self.Match.CtLabelHi != 0 || self.Match.CtLabelLo != 0 {
+	if self.Match.CtLabelHiMask != 0 || self.Match.CtLabelLoMask != 0 {
```

Comment:
I'm not sure if I understand the change. The check is same as the one in L454, which means at least one of them is wrong.
@wenyingd

Reply:
L450 will check ct_label value non-zero to process ct_label field. This will miss ct_label as zero case. Actually, ct_label mask must not be zero if any ct_label bit is used. Here, we can use ct_label mask as check condition to process ct_label.

Reply:
I mean with this change, L454 is true for sure and L460 will never be called

Reply:
Yeah, I also considered it when changed code. L460 may be removed or just leave it there, there is no bad side effect after all.

Reply:
Mask for ct_label is not always set even if we have set ct_label value, in this case we actually want to leverage the complate ct_label for a value. With your change, we can't cover this case. 

I guess you want to cover the case that ct_label is 0, but mask for ct_label is set as an anti-flag for the original connection identifier. I would suggest you checking both ct_lable and mask values.

Reply:
Thanks @wenyingd, you are right. The ct_labels may be set with zero mask. If we want to cover zero mask and zero labels, we need to check both labels and mask value. I have updated the patch. Please help review again.